tm2/pkg/crypto/keys/client/import.go

```diff
@@ -121,10 +122,16 @@ func execImport(cfg *ImportCfg, io commands.IO) error {
 	}
 
 	if cfg.Unsafe {
+		// Un-armor the private key
+		privKey, err := armor.UnarmorPrivateKey(string(keyArmor))
+		if err != nil {
+			return fmt.Errorf("unable to unarmor private key, %w", err)
+		}
+
 		// Import the unencrypted private key
-		if err := kb.ImportPrivKeyUnsafe(
+		if err := kb.ImportPrivKey(
```

Comment:
These two blocks are also duplicated (132-141) and (150-159). You could move them after the if-else.

Reply:
Good catch 💯 

These "features" is something I'll change in a separate PR 🙏 

[3acb4c6](https://github.com/gnolang/gno/pull/3285/commits/3acb4c690b8e302fad52e57a6e04c27b1393d024)